### DISCUSSION TITLE
Scratch addon suggestion

### DISCUSSION THREAD

**Discussion by LilSefGaming** [ScratchAddons/ScratchAddons#4145] [Ideas] [answered]
TurboWarp blocks addons!
:)

**Comment by Dhuls-Scratch** [comment, marked as answer]
for the <is compiled?> and <is Turbowarp?> blocks just backpack them from [here](https://scratch.mit.edu/projects/414716080/).
The other blocks would cause issues for those who don't use Scratch Addons.

**Comment by LilSefGaming** [comment]
Aight thx
